org-formation/090-systems-manager/_tasks.yaml

```diff
@@ -60,3 +60,17 @@ PatchMembers:
     IncludeMasterAccount: true
   Parameters:
     ManagementAccountNumber: !Ref accountId
+
+StackArmorNessusScanner:
+  Type: update-stacks
+  Template: ScheduledMaintenance.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-stackarmor-nessus-scsnner'
+  StackDescription: Install Nessus scanner on instances in account
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Accounts:
+      - !Ref SynapseProdAccount
```

Comment:
i think running from the central `SysmanCentralAccount` would be better for organizing runbooks.  it also allows running scripts from a central location in multiple accounts in the organization.

Reply:
I'm not sure what it means to "run" an SSM Maintenance window from a central account. Can you please explain?

Reply:
`SysmanCentralAccount` is a delegated administrator account for systems manager feature which means that it has access to all instances in all member AWS accounts in our organization.  The way to run automation across multiple accounts is documented in https://docs.aws.amazon.com/systems-manager/latest/userguide/running-automations-multiple-accounts-regions.html

This is how we are able to setup one central aws account to do OS patching to instances in all accounts.   You may want to take another look at the [templates we used to setup patch manager and associated roles](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/090-systems-manager/_tasks.yaml).

Reply:
SysmanCentralAccount is a delegated administrator account for systems manager feature which means that it has access to all instances in all member AWS accounts in our organization. The way to run automation across multiple accounts is documented in https://docs.aws.amazon.com/systems-manager/latest/userguide/running-automations-multiple-accounts-regions.html

This is how we are able to setup one central aws account to do OS patching to instances in all accounts. You may want to take another look at the [templates we used to setup patch manager and associated roles](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/090-systems-manager/_tasks.yaml).

Reply:
The AWS doc' you reference says:

> When you run an automation across multiple Regions and accounts, you target resources by using tags or the name of an AWS resource group. The resource group must exist in each target account and Region. The resource group name must be the same in each target account and Region. The automation fails to run on those resources that don't have the specified tag or that aren't included in the specified resource group.

So this approach can't run on all instances in a selected account by default.  The instances would have to be explicitly tagged or else be explicitly added to a resource group. Is this what we do to make patch manager work on all instances, tag or group the instances to patch?

Reply:
Can we setup a resource group based only on the type of resource (AWS::Ec2::Instances) with no tag? From what I see the patchManager depends on 2 tags (present in scipoolprod, but not in synapseprod).

Reply:
> Is this what we do to make patch manager work on all instances, tag or group the instances to patch?

yes, we setup a resource group to target where to run the patches.  The definition of the patch resource group is here  https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/090-systems-manager/patch-resource-group.yaml